pkgs/build-support/add-driver-runpath/default.nix

```diff
@@ -0,0 +1,14 @@
+{ lib, stdenv }:
+
+stdenv.mkDerivation {
+  name = "add-driver-runpath";
+
+  # Named "opengl-driver" for legacy reasons, but it is the path to
+  # hardware drivers installed by NixOS
```

Comment:
IIRC previously there was an initiative to renamed this path. perhaps that's worth revisiting at some point?